### PR TITLE
Add the inducer to mining & survey vendors.

### PR DESCRIPTION
**Pull request by lbnesquik**
They are relatively expensive so it is still better to bother competent R&D.

### CODE REVIEW

**Comment by lm40** [comment]
to be honest i'm not 100% sure what they would be used for but sure why not

File: code/modules/mining/ore_redemption_machine/equipment_vendor.dm [hunk 1]
```diff
@@ -91,8 +91,9 @@
 		EQUIPMENT("Fine Excavation Kit - Measuring Tape",	/obj/item/device/measuring_tape,				125),
 		EQUIPMENT("Fine Excavation Kit - Hand Pick",		/obj/item/weapon/pickaxe/hand,					375),
 		EQUIPMENT("Explosive Excavation Kit - Plastic Charge",/obj/item/weapon/plastique/seismic/locked,	1500),
-		EQUIPMENT("Industrial Equipment - Phoron Bore",	/obj/item/weapon/gun/magnetic/matfed,				3000),
-		EQUIPMENT("Industrial Equipment - Sheet-Snatcher",/obj/item/weapon/storage/bag/sheetsnatcher,		500),
+		EQUIPMENT("Industrial Equipment - Phoron Bore",		/obj/item/weapon/gun/magnetic/matfed,			3000),
+		EQUIPMENT("Industrial Equipment - Inducer",			/obj/item/weapon/inducer,						900),
```
Comment (Novacat) [inline code/modules/mining/ore_redemption_machine/equipment_vendor.dm:95]: General rule of thumb is that the Mining prices are 10x the Survey prices

Comment (lbnesquik) [reply code/modules/mining/ore_redemption_machine/equipment_vendor.dm:95]: Ok

Comment (lbnesquik) [reply code/modules/mining/ore_redemption_machine/equipment_vendor.dm:95]: Fixed

**Comment by nerdass** [comment]
Don't really see anything bad about this since it's adding the one that specifically CAN'T recharge guns.

**Comment by JoanRisu** [comment]
I concur with K't. Unless I'm missing something, I think we're good on this one.

**Pull request merged**